test_scripts/MobileProjection/Phase2/common.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local test = require("user_modules/dummy_connecttest")
+local events = require("events")
+
+--[[ Module ]]
+local m = actions
+
+m.failedTCs = {}
+
+m.wait = utils.wait
+
+function m.setAppConfig(pAppId, pAppHMIType, pIsMedia)
+  m.getConfigAppParams(pAppId).appHMIType = { pAppHMIType }
+  m.getConfigAppParams(pAppId).isMediaApplication = pIsMedia
+end
+
+function m.registerApp(pAppId)
```

Comment:
Maybe can be used registerAppWOPTU from actions.lua

Reply:
@GetmanetsIrina Done in b0c29ef